src/main/java/systems/composable/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -451,6 +471,7 @@ public Cluster build(MetricRegistry metrics, HealthCheckRegistry healthChecks) {
             builder.addContactPoints(contactPoint);
         }
 
+
```

Comment:
Please don't add extra whitespace. The PR should conform to the existing code styles used.